app/angular/login/login.html

```diff
@@ -16,38 +16,43 @@ <h1>BR Modelo</h1>
 			</div>
 			<form class="login-form" name="loginForm" ng-submit="$ctrl.submitForm(loginForm.$valid)" novalidate>
 				<div class="form-group">
-					<label for="exampleInputEmail1">Email</label>
+					<label for="exampleInputEmail1">{{ "Email" | translate }}</label>
 					<input
 						type="email"
 						name="email"
 						class="form-control input-lg"
 						id="userEmail"
-						placeholder="Email"
+						placeholder="{{ 'Email' | translate }}"
 						autocomplete="off"
 						required
 						ng-model="$ctrl.credentials.username"
 						data-ng-class="{'error': loginForm.email.$invalid && (!loginForm.email.$pristine || $ctrl.submitted)}"
 					/>
 				</div>
 				<div class="form-group">
-					<label for="exampleInputPassword1">Password</label>
+					<label for="exampleInputPassword1">{{ "Password" | translate }}</label>
```

Comment:
```suggestion
					<label for="exampleInputPassword1">{{ 'Password'| translate }}</label>
```

You're using single quotes in the other places, I think you can change here to keep a pattern

Reply:
Done!